identify/extensions.py

```diff
@@ -16,6 +16,7 @@
     'cc': {'text', 'c++'},
     'cu': {'text', 'cuda'},
     'cfg': {'text'},
+    'chs': {'text', 'c2hs'},
```

Comment:
Im not totally happy with the lack of symmetry with the other 2 haskell variants below, but any variant of 'c2-haskell', 'c-2-haskell', 'c-to-haskell' just looked off to me